src/subscription_manager/repofile.py

```diff
@@ -610,6 +619,10 @@ def fix_content(self, content):
         if zypper_cont['gpgkey'] in ['https://', 'http://']:
             del zypper_cont['gpgkey']
 
+        # make sure gpg key download doesn't fail because of private certs
+        if zypper_cont['gpgkey'] and self.gpgkey_ssl_verify
```

Comment:
I think that colon is missing at the end of this line.

Reply:
thanks!